sources/packages/forms/src/form-definitions/sfaa2024-25.json

```diff
@@ -2673,19 +2673,18 @@
                         }
                       ],
                       "className": "alert alert-warning fa fa-exclamation-triangle w-100",
-                      "content": "<span class=\"font-bold ml-2\">Study period not eligible for funding</span>\n<br>\n<span>The study period you selected has already concluded and will need additional review from StudentAid BC to be considered for funding. Once you have submitted your application, please use the following link for information on the <a class=\"formio-href\" href=\"https://studentaidbc.ca/sites/all/files/form-library/appeal_fundingafterenddate.pdf\" target=\"_blank\" rel=\"noopener noreferrer\">funding after end date appeal</a>.</span>",
+                      "content": "<span class=\"font-bold ml-2\">Study period not eligible for funding</span>\n<br>\n<span>The dates you have entered are no longer eligible for funding. To receive funding for this study period, you will need to seek approval from StudentAid BC. Once you have submitted your application, please use the following link for information on the <a class=\"formio-href\" href=\"https://studentaidbc.ca/sites/all/files/form-library/appeal_fundingafterenddate.pdf\" target=\"_blank\" rel=\"noopener noreferrer\">funding after end date appeal</a>.</span>",
```

Comment:
Can the code be simplified as below?
```js
value = !!data.studyendDate && moment(data.studyendDate).isBefore(moment().subtract(6, 'week'));
````
Also, `studyendDate` is timezone agnostic. Is there a reason UTC was in the logic?